### PR TITLE
[Workers] Profiling Workers revision

### PR DESCRIPTION
**Pull request by deadlypants1973**
do not merge

### CODE REVIEW

File: products/workers/src/content/learning/profiling-workers.md [hunk 1]
```diff
@@ -5,13 +5,13 @@ pcx-content-type: concept
...
 
-For long-running or complex Workers, `wrangler inspect`(beta) will help you understand where your code is spending the most time.
+For long-running or complex Workers, `wrangler inspect`(beta) will help you understand what part of your code takes the longest time to execute.
 
 As part of the Workers platform's integration with Chrome Developer Tools, you can use `wrangler dev --inspect` to get a CPU profile of your Worker.
 
 <StreamVideo id="f11809a382160334e9be9a2aedf13d1d" />
 
-To run `wrangler inspect`, first, make sure you are on Wrangler version v1.19.3 or later. If you need to update your version, refer to the guide on [upgrading Wrangler](https://developers.cloudflare.com/workers/cli-wrangler/install-update).
+`wrangler inspect` will only work on Wrangler version v1.19.3 or later. Check your version before continuing and if you need to update your version, refer to the guide on [upgrading Wrangler](https://developers.cloudflare.com/workers/cli-wrangler/install-update).
```
Comment (marciocloudflare) [inline products/workers/src/content/learning/profiling-workers.md]:
```suggestion
`wrangler inspect` will only work on Wrangler version v1.19.3 or later. Check what version you have before continuing. If you need to update Wrangler, refer to the guide on [upgrading Wrangler](https://developers.cloudflare.com/workers/cli-wrangler/install-update).
```

Comment (marciocloudflare) [reply products/workers/src/content/learning/profiling-workers.md]: Trying to cut the long sentence into smaller ones

File: products/workers/src/content/learning/profiling-workers.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ This is a basic example where the Worker's request handler calls an async functi
 
 ### Profiling our Worker
 
-With this function saved, go to your terminal and run `wrangler dev --inspect`. You will see additional instructions for configuring Chrome DevTools.
+With your `index.js` file updates and saved, go to your terminal and run `wrangler dev --inspect`. You will see additional instructions for configuring Chrome DevTools.
```
Comment (marciocloudflare) [inline products/workers/src/content/learning/profiling-workers.md]:
```suggestion
With your `index.js` file updated and saved, go to your terminal and run `wrangler dev --inspect`. You will see additional instructions for configuring Chrome DevTools.
```

File: products/workers/src/content/learning/profiling-workers.md [hunk 3]
```diff
@@ -63,21 +63,21 @@ $ wrangler dev --inspect
 👂  Listening on http://127.0.0.1:8787
 ```
 
-Open a new tab in Chrome, type in `chrome://inspect` and hit enter. Select **Configure** and add `localhost:9230`. The Wrangler inspect process will then show up under **Remote Target**.
+Open a new tab in Chrome, type `chrome://inspect` in the Chrome address bar and press **Enter** or **return** on your keyboard to go to Chrome's DevTools. Select **Configure...** and add `localhost:9230`. The Wrangler inspect process will then show up under **Remote Target**.
```
Comment (marciocloudflare) [inline products/workers/src/content/learning/profiling-workers.md]:
```suggestion
Open a new tab in Chrome and enter `chrome://inspect` in the Chrome address bar to open Chrome's DevTools. Select **Configure...** and add `localhost:9230`. The Wrangler inspect process will then show up under **Remote Target**.
```

Comment (marciocloudflare) [reply products/workers/src/content/learning/profiling-workers.md]: Wdyt?

**Review by marciocloudflare — APPROVED** [review]

File: products/workers/src/content/learning/profiling-workers.md [hunk 4]
```diff
@@ -5,13 +5,13 @@ pcx-content-type: concept
 
 # Profiling Workers
 
-For long-running or complex Workers, `wrangler inspect`(beta) will help you understand where your code is spending the most time.
+For long-running or complex Workers, `wrangler inspect`(beta) will help you understand what part of your code takes the longest time to execute.
```
Comment (lukeed) [inline products/workers/src/content/learning/profiling-workers.md]:
```suggestion
For long-running or complex Workers, `wrangler inspect` (beta) will help you understand which parts of your code require the most time to execute.
```

File: products/workers/src/content/learning/profiling-workers.md [hunk 5]
```diff
@@ -5,13 +5,13 @@ pcx-content-type: concept
...
 
 As part of the Workers platform's integration with Chrome Developer Tools, you can use `wrangler dev --inspect` to get a CPU profile of your Worker.
 
 <StreamVideo id="f11809a382160334e9be9a2aedf13d1d" />
 
-To run `wrangler inspect`, first, make sure you are on Wrangler version v1.19.3 or later. If you need to update your version, refer to the guide on [upgrading Wrangler](https://developers.cloudflare.com/workers/cli-wrangler/install-update).
+`wrangler inspect` will only work on Wrangler version v1.19.3 or later. Check what version you have before continuing. If you need to update Wrangler, refer to the guide on [upgrading Wrangler](https://developers.cloudflare.com/workers/cli-wrangler/install-update).
```
Comment (lukeed) [inline products/workers/src/content/learning/profiling-workers.md]:
```suggestion
`wrangler inspect` is only available in Wrangler v1.19.3 or later. Check what version you have before continuing. If you need to update Wrangler, refer to the guide on [upgrading Wrangler](https://developers.cloudflare.com/workers/cli-wrangler/install-update).
```

File: products/workers/src/content/learning/profiling-workers.md [hunk 6]
```diff
@@ -63,21 +63,21 @@ $ wrangler dev --inspect
...
 
-Currently, Wrangler only supports the Console, Sources and Profiler tabs.
+Currently, Wrangler only supports the **Console**, **Sources** and **Profiler** tabs.
 
 </Aside>
 
 ### Capturing a CPU profile
 
-With the DevTools open, select the **Profile** tab > **Start**. Open a new tab with your Worker running locally `http://127.0.0.1:8787`. Visiting this address will invoke the Worker again, allowing DevTools to capture execution information. When it finishes loading, go back to our DevTools window and select **Stop**.
+With the DevTools open, select the **Profile** tab > **Start**. Open a new tab with your Worker running locally `http://127.0.0.1:8787`. Visiting this address will invoke the Worker again, and allow DevTools to capture execution information. When it finishes loading, go back to the DevTools window and select **Stop**.
```
Comment (lukeed) [inline products/workers/src/content/learning/profiling-workers.md]:
```suggestion
With the DevTools open, select the **Profile** tab > **Start**. Open a new tab with your Worker running locally `http://127.0.0.1:8787`. Visiting this address will invoke the Worker again and allow DevTools to capture execution information. When it finishes loading, go back to the DevTools window and select **Stop**.
```

**Pull request merged**